frontends/web/src/routes/buy/components/exchangeselectionradio.module.css

```diff
@@ -29,14 +29,22 @@
 }
 
 .paymentMethodName {
-    width: 50%;
+    color: var(--color-dark);
```

Comment:
not yet, but soon we should not use color variables directly, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/8d853d1c98aa540169f995d26f8b79ddc06f885a/frontends/web/src/style/variables.css#L23-L25

(I forgot to check in the last PR)

Reply:
Sure. Changed all in this file but one, the `--color-skyblue` - as it's only used once in this entire app and we didn't define its "background colour" name yet.


> not yet, but soon we should not use color variables directly,

If so,  we should start thinking on defining the "names" (bg, font, status) for all these new colours that were introduced and are only used in this "buy" page. Bc atm, we're using it directly, e.g `--color-olive` or `--color-light-skyblue`,etc..